test/templates/feature_9/Registry.toml

```diff
@@ -0,0 +1,8 @@
+name = "ExampleRegistry"
```

Comment:
this is a copy of `feature_1`, but with `Requires` changed to `Requires-dash` everywhere